--- android_icu4j/src/main/java/android/icu/util/Calendar.java	2026-01-23 12:08:06.202669979 +0000
+++ android_icu4j/src/main/java/android/icu/util/Calendar.java	2026-01-23 12:08:34.540109565 +0000
@@ -4652,8 +4652,6 @@
      * @param date the date and time
      * @return true if the given date and time is part of the
      * weekend
-     * @see #getDayOfWeekType
-     * @see #getWeekendTransition
      * @see #isWeekend()
      */
     public boolean isWeekend(Date date) {
@@ -4666,8 +4664,6 @@
      * this calendar system.
      * @return true if the given date and time is part of the
      * weekend
-     * @see #getDayOfWeekType
-     * @see #getWeekendTransition
      * @see #isWeekend(Date)
      */
     public boolean isWeekend() {
