--- android_icu4j/src/main/java/android/icu/number/NumberFormatterSettings.java	2026-01-13 12:28:59.828695390 +0000
+++ android_icu4j/src/main/java/android/icu/number/NumberFormatterSettings.java	2026-01-13 12:29:34.615743636 +0000
@@ -124,19 +124,10 @@
      * NumberFormatter.with().unit(Currency.getInstance("USD"))
      * </pre>
      *
-     * Percent:
-     *
-     * <pre>
-     * NumberFormatter.with().unit(NoUnit.PERCENT)
-     * </pre>
-     *
      * <p>
      * See {@link #perUnit} for information on how to format strings like "5 meters per second".
      *
      * <p>
-     * The default is to render without units (equivalent to {@link NoUnit#BASE}).
-     *
-     * <p>
      * If the input usage is correctly set the output unit <b>will change</b>
      * according to {@code usage}, {@code locale} and {@code unit} value.
      * </p>
@@ -146,7 +137,6 @@
      * @return The fluent chain.
      * @see MeasureUnit
      * @see Currency
-     * @see NoUnit
      * @see #perUnit
      */
     public T unit(MeasureUnit unit) {
