--- android_icu4j/src/main/java/android/icu/text/Collator.java	2026-03-19 14:48:57.536597912 +0000
+++ android_icu4j/src/main/java/android/icu/text/Collator.java	2026-03-19 14:49:22.557540374 +0000
@@ -1228,7 +1228,6 @@
      *         CollationKey is returned.
      * @see CollationKey
      * @see #compare(String, String)
-     * @see #getRawCollationKey
      */
     public abstract CollationKey getCollationKey(String source);
 
