--- android_icu4j/src/main/java/android/icu/text/Collator.java	2026-01-13 12:28:55.078810132 +0000
+++ android_icu4j/src/main/java/android/icu/text/Collator.java	2026-01-13 12:29:34.655750284 +0000
@@ -1228,7 +1228,6 @@
      *         CollationKey is returned.
      * @see CollationKey
      * @see #compare(String, String)
-     * @see #getRawCollationKey
      */
     public abstract CollationKey getCollationKey(String source);
 
