--- android_icu4j/src/main/java/android/icu/text/Collator.java	2026-01-13 18:29:17.192779973 +0000
+++ android_icu4j/src/main/java/android/icu/text/Collator.java	2026-01-13 18:29:54.391718857 +0000
@@ -1228,7 +1228,6 @@
      *         CollationKey is returned.
      * @see CollationKey
      * @see #compare(String, String)
-     * @see #getRawCollationKey
      */
     public abstract CollationKey getCollationKey(String source);
 
