--- android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2026-01-23 12:08:00.504875318 +0000
+++ android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2026-01-23 12:08:34.320208311 +0000
@@ -82,7 +82,6 @@
  *
  * @see          DateFormat
  * @see          SimpleDateFormat
- * @see          android.icu.util.SimpleTimeZone
  * @author       Chen-Lieh Huang
  */
 public class DateFormatSymbols implements Serializable, Cloneable {
