--- android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2026-01-13 18:29:22.678456141 +0000
+++ android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2026-01-13 18:29:54.431249018 +0000
@@ -82,7 +82,6 @@
  *
  * @see          DateFormat
  * @see          SimpleDateFormat
- * @see          android.icu.util.SimpleTimeZone
  * @author       Chen-Lieh Huang
  */
 public class DateFormatSymbols implements Serializable, Cloneable {
